src/webhooks/models/webhooksFormatter.js

```diff
@@ -65,7 +73,7 @@ function json(event, testId, jobId, report, additionalInfo, options) {
     return payload;
 }
 
-function slack(event, testId, jobId, report, additionalInfo, options) {
+function slack(event, format, testId, jobId, report, additionalInfo, options) {
```

Comment:
I do not think `slack`, `teams` and `discord` should take format as an argument, each function should pass the proper value to `emojiHandler`